catalog/nsili/catalog-nsili-endpoint/src/main/java/org/codice/alliance/nsili/endpoint/requests/SubmitStandingQueryRequestImpl.java

```diff
@@ -202,45 +200,44 @@ public String getId() {
   }
 
   @Override
-  public void set_number_of_hits(int hits)
```

Comment:
❗️ You can't rename these methods. They override interface methods in generated CORBA api code.